src/controllers/user.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+const bcrypt = require('bcrypt');
+const registerValidation = require('../validation/user');
+const User = require('../services/user');
+const { successResponse, handleError, errorResponse } = require('../utils/response');
+
+/**
+ * @class UserController
+ * @description create, verify and log in user
+ * @exports UserController
+ */
+module.exports = class UserController {
+  /**
+   * @param {object} req - The user request object
+   * @param {object} res - The user response object
+   * @returns {object} Success message
+   */
+  static async createUser(req, res) {
+    try {
+      const { error } = registerValidation(req.body);
+      if (error) {
+        return res.status(400).json({ status: 400, error: error.message });
```

Comment:
@Osasbaby Let your logic be in your service folder

Reply:
done